src/MongoDB/Exception/RuntimeException.c

```diff
@@ -23,12 +23,90 @@
 
 #include "phongo_compat.h"
 #include "php_phongo.h"
+#include "php_array_api.h"
 
 zend_class_entry* php_phongo_runtimeexception_ce;
 
+static bool php_phongo_has_string_array_element(zval* labels, char* label TSRMLS_DC)
+{
+	HashTable* ht_data;
+
+	if (Z_TYPE_P(labels) != IS_ARRAY) {
+		return false;
```

Comment:
I pondered whether a LogicException or something similar would work here, but it's probably best to keep `hasErrorLabel()` as simple as possible. The edge case I was thinking of was if someone uses reflection to re-assign our private property.

Reply:
Added

Reply:
What was added? I wasn't suggesting actually throwing, just explaining my thought process. I do think it's preferable to just return false here.

Reply:
Added "added" to the wrong place I think. Want to say I added the test cases, perhaps? /me shrugs